modules/apps/frontend-js/frontend-js-alert-support-web/npmscripts.config.js

```diff
@@ -14,7 +14,6 @@
 
 module.exports = {
 	federation: {
-		runLegacyBuild: false,
+		mode: 'default',
```

Comment:
Similar question here. Wouldn't `federation: true` imply the `'default'` should be in effect?